src/Networks/Blockcore.Networks.XRC/XRCMain.cs

```diff
@@ -156,16 +139,26 @@ public XRCMain()
             this.Base58Prefixes[(int)Base58Type.CONFIRMATION_CODE] = new byte[] { 0x64, 0x3B, 0xF6, 0xA8, 0x9A };
             this.Base58Prefixes[(int)Base58Type.ASSET_ID] = new byte[] { 23 };
 
-            this.Checkpoints = new Dictionary<int, CheckpointInfo>
-            {
-                { 17,new CheckpointInfo(new uint256("2430c4151e10cdc5ccbdea56b909c7c37ab2a852d3e7fb908e0a32493e2ac706")) },
-                { 117, new CheckpointInfo(new uint256("bf3082be3b2da88187ebeb902548b41dbff3bcac6687352e0c47d902acd28e62"))},
-                { 400, new CheckpointInfo(new uint256("20cb04127f12c1ae7a04ee6dc4c7e36f4c85ee2038c92126b3fd537110d96595"))},
-                { 800, new CheckpointInfo(new uint256("df37ca401ecccfc6dedf68ab76a7161496ad93d47c2a474075efb3220e3f3526"))},
-                { 26800, new CheckpointInfo(new uint256("c4efd4b6fa294fd72ab6f614dd6705eea43d0a83cd03d597c3214eaaf857a4b6"))},
-                { 43034, new CheckpointInfo(new uint256("4df06bd483d2c4ccde5cd1efe3b2ea7d969c41e5923a74c2bba1656a41fc6891"))},
-                { 110000, new CheckpointInfo(new uint256("d1d1282681f20223a281393528e6c624539e60177ecb42ab4512555974ac7775"))},
-            };
+            this.Checkpoints = new Dictionary<int, CheckpointInfo>();
+            this.Checkpoints.Add(2015, new CheckpointInfo(new uint256("574605587514315bf8dac135c093a50e5982cb26e47ac78f2a712b9289f5cc7e")));
```

Comment:
Replacing all the old checkpoints is not acceptable. Please include the old checkpoints.

Reply:
Ok it is back in https://github.com/block-core/blockcore/pull/402/commits/a5cf88709467fb1cd14529addacc3f00b0d5d4f7 :)